tools/sdk/uxdProtocol/createInitializeControllerInstruction.ts

```diff
@@ -4,13 +4,16 @@ import { uxdClient } from './uxdClient'
 
 const createInitializeControllerInstruction = (
   uxdProgramId: PublicKey,
-  mintSymbol: string,
   mintDecimals: number,
   authority: PublicKey,
   payer: PublicKey,
   connection: Connection
 ): TransactionInstruction => {
-  const { client, controller } = uxdClient(connection, uxdProgramId)
```

Comment:
This is just for the time being, but that's not what we want. Controller has nothing to do with uxd client. 
The front should have several layer, but in the highest layer where we do business logic, the only things that should emerge are "depository, controller, client, mango", the point of these objects is to abstract complexity.
Instantiating them at the last moment just to match the API format defeat the purpose